Tests/Shared.Specs/Xml/XDocumentAssertionSpecs.cs

```diff
@@ -56,6 +56,21 @@ public void When_asserting_a_null_xml_document_is_equal_to_another_xml_document_
             act.Should().Throw<XunitException>();
         }
 
+        [Fact]
+        public void When_a_a_null_xml_document_is_equal_to_another_null_xml_document_it_should_fail()
```

Comment:
`When_asserting_a_null_xml_document_is_equal_to_another_a_null_xml_document_it_should_succeed`

or your implemented assertion is wrong.

Reply:
Thanks